emscripten.py

```diff
@@ -581,6 +590,14 @@ def read_proxied_function_signatures(asmConsts):
   if not shared.Settings.WASM_BACKEND:
     shared.Settings.PROXIED_FUNCTION_SIGNATURES = read_proxied_function_signatures(metadata['asmConsts'])
 
+  shared.Settings.STATIC_BUMP = align_static_bump(metadata)
+
+
+def apply_forwarded_data(forwarded_data):
+  forwarded_json = json.loads(forwarded_data)
+  # Be aware of JS static allocations
+  shared.Settings.STATIC_BUMP = forwarded_json['STATIC_BUMP']
```

Comment:
It is slightly odd, yeah, we use `shared.Settings` as a global location for compiler state. We do mark them in `settings.js` with comments saying some are just for internal use, like this one. But perhaps they could be in `shared.Internals` or such?

Specifically here, this can't be a local var to the file since we do want to use it elsewhere (e.g. when we write the metadata we write this - although I didn't update all those places yet in this PR, to keep it small).